### PR TITLE
[UI] Fix small issues generating console errors

### PR DESCRIPTION
**Pull request by prymitive**

### CODE REVIEW

File: web/ui/react-app/src/pages/alerts/AlertContents.tsx [hunk 1]
```diff
@@ -55,8 +55,8 @@ const AlertsContent: FC<AlertsProps> = ({ groups = [], statsCount }) => {
     });
   };
 
-  const toggleAnnotations = () => {
-    setShowAnnotations({ checked: !showAnnotations.checked });
+  const toggleAnnotations = (checked: boolean) => {
```
Comment (juliusv) [inline web/ui/react-app/src/pages/alerts/AlertContents.tsx:58]: I think at this point you can delete this function and better inline it :smile:

Comment (prymitive) [reply web/ui/react-app/src/pages/alerts/AlertContents.tsx]: good point, will do

**Comment by juliusv** [comment]
:+1: Thanks!

**Comment by roidelapluie** [comment]
@prymitive do you happen to have paid circleci AND netfify subscriptions that would make those CI builds not to run?

**Comment by prymitive** [comment]
I do not, I did create a CircleCI account a while ago but never really used it, so I revoked it from GH, maybe that's the problem.
I've logged into CircleCI, which required me to allow it access my github data and that seemed to fix it.

**Comment by roidelapluie** [comment]
Thank you for finding out. Now it seems that UI tests are failing ;)

**Comment by prymitive** [comment]
Yes, sorry about that, should be fixed now

**Comment by roidelapluie** [comment]
I have tried a fix for netlify..

**Comment by roidelapluie** [comment]
Thanks @prymitive !

**Pull request merged**